packages/aws-client/tests/test-S3.js

```diff
@@ -377,3 +379,33 @@ test('getObjectSize() returns the size of an object', async (t) => {
 
   t.is(objectSize, 4);
 });
+
+test('streamS3Upload() uploads contents of stream to S3', async (t) => {
```

Comment:
Should we have tests here for the error/reject paths?

Reply:
520a30525fe721fbae304eb0e99318adee9dfe03